src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int LIMITING_NUMBER = 101;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(),
+                new Random().nextInt(LIMITING_NUMBER));
```

Comment:
For Random and ColorSupplier:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/d2c56165-7541-4e71-a31c-75bad41c1179)